homeassistant/components/starline/button.py

```diff
@@ -0,0 +1,74 @@
+"""Support for StarLine button."""
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from homeassistant.components.button import ButtonEntity, ButtonEntityDescription
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .account import StarlineAccount, StarlineDevice
+from .const import DOMAIN
+from .entity import StarlineEntity
+
+
+@dataclass
+class StarlineRequiredKeysMixin:
+    """Mixin for required keys."""
+
+    name_: str
+
+
+@dataclass
+class StarlineButtonEntityDescription(
+    ButtonEntityDescription, StarlineRequiredKeysMixin
+):
+    """Describes Starline button entity."""
+
+
+BUTTON_TYPES: tuple[StarlineButtonEntityDescription, ...] = (
+    StarlineButtonEntityDescription(
+        key="poke",
+        name_="Horn",
```

Comment:
Who we use `name_`? Why don't we just use a translation key

Reply:
My mistake, fixed